package.json

```diff
@@ -59,6 +59,7 @@
     "react-dom": "^15.3.2",
     "react-redux": "^5.0.1",
     "react-router": "^3.0.0",
+    "react-router-redux": "^4.0.8",
```

Comment:
could we also update the [`yarn.lock` file](https://yarnpkg.com/en/docs/yarn-lock)?